mem-03-D.xml

```diff
@@ -4329,7 +4362,9 @@ Watch Gowron say this: {YouTube video:url:http://youtu.be/7qXuTn9HpP4}</column>
       <column name="examples_de"></column>
       <column name="examples_fa"></column>
       <column name="examples_sv"></column>
-      <column name="examples_ru"></column>
+      <column name="examples_ru">
+▶ {Dochvetlh DIlmeH Huch 'ar DaneH?:sen} "Сколько ты хочешь за это?"[1]
```

Comment:
This should not have been translated here. The other examples were also removed.

I've filed a bug to improve the handling of examples (which will, again, probably have to wait for a database redesign from scratch):
https://github.com/De7vID/klingon-assistant/issues/463